### ISSUE TITLE
Documentation improvement

### ISSUE THREAD

**Issue by jrmlstf** [picanteverde/init#1]
Maybe you should use Init.js or Initjs (or just Init with an uppercase i) in order to limit confusion/improve comprehension while init is a commonly used function name.

These lines of the documentation perfectly exposes what it mean :

> git init
> Next lets add a git remote with the init repo
> git remote add init git://github.com/picanteverde/init.git

Perhaps i'm too much ? It's just a suggestion...

**Comment by picanteverde** [comment]
Yes I now, maybe Initjs is the best using init as a project name also makes me invisible in google search. What do you think, should I change the name of the repo (and break all current links)? or start a new repo with the name Initjs and redirect to it?

**Comment by jrmlstf** [comment]
I'm not an expert. Couldn't say what is the best option. Can't we find other projects which had to solve this kind of issue ?

**Comment by msaglietto** [comment]
If you are planing to maintain the project my suggestion is create a init.js (or the name of your chose) and redirect this project to that ... update the post on toptal and your blog .. and after a while just delete this repo

**Comment by gabhi** [comment]
@picanteverde,  I think if you are stuck at renaming the repository following link may help you
https://help.github.com/articles/renaming-a-repository
